manager.spec.yaml

```diff
@@ -783,6 +824,31 @@ paths:
             application/vnd.api+json:
               schema:
                 $ref: '#/components/schemas/CvesWithoutErrataOut'
+patch:
```

Comment:
This is incorrectly formatted, only lines 805-806 in this file should be changed.